ewebsock/src/lib.rs

```diff
@@ -133,12 +133,24 @@ pub struct Options {
     ///
     /// Ignored on Web.
     pub max_incoming_frame_size: usize,
+
+    /// Additional Request headers
+    pub additional_headers: Option<Vec<(String, String)>>,
+
+    /// Additional subprotocols
+    pub subprotocols: Option<Vec<String>>,
+
+    /// Delay blocking in ms - default 10ms
+    pub delay_blocking: Option<std::time::Duration>,
```

Comment:
The default would be more apparent if you removed the `Option` and just set the 10 ms in the `default()`